content/applications/websites/livechat/chatbots.rst

```diff
@@ -121,9 +122,9 @@ Forward to Operator
 *******************
 
 This step forwards the conversation to an active live chat operator, so that they can continue
-assisting the visitor. As the conversation transcript is passed on to the operator they are able to
-pick up where the chatbot left off. This not only saves time for all parties involved, it can also
-help qualify conversations before they reach live human operators.
+assisting the visitor. As the conversation transcript is passed on to the operator, they can pick up
+where the chatbot left off. This not only saves time for all parties involved, it can also help
+qualify conversations before they reach live operators.
 
 .. note::
    If no active operator is available on the channel, the chatbot continues the conversation with
```

Comment:
Just a suggestion- but you could add the message from the image in this note. Because when I read 'additional steps' i thought of more questions and answers, not an 'everyone is busy'. Again, not necessary, just a suggestion.